Include default typoscript installation-wide

dimitri-koenig opened the pull request:
Currently the dashboard template cannot be loaded, see #35. This fixes it.

Temporary workaround: patch via composer-patches:

```
        "pluswerk/mail-logger": {
            "Fix backend view path": "https://github.com/dimitri-koenig/mail_logger/pull/1.patch"
        }
```

Fixes #35

Kanti commented:
Instead of this workaround, you should probably include the TypoScript of this extension in your Sites TypoScript

Add `@import 'EXT:mail_logger/Configuration/TypoScript/setup.typoscript'` in your setup
And `@import 'EXT:mail_logger/Configuration/TypoScript/constant.typoscript'` in your constants.

dimitri-koenig commented:
That would work for Frontend, but not for Backend, wouldn't it?

dimitri-koenig commented:
I've changed the code, removed the initializeView part, and added an ext_localconf.php file which includes the typoscript installation-wide. I had some warnings when opening up the backend module. This fixes it.

josefglatz commented:
My workaround was also to add the proper TypoScript in our SitePackage

Once this get's merged I can remove my workaround in the project.

Just for the reference, I had to provide key, type and content:

```php
\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addTypoScript(
    'mail_logger',
    'setup',
    '@import \'EXT:mail_logger/Configuration/TypoScript/setup.typoscript\''
);
\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addTypoScript(
    'mail_logger',
    'constants',
    '@import \'EXT:mail_logger/Configuration/TypoScript/constants.typoscript\''
);

```

Kanti commented:
I had time to test it with our Projects, did does not break anything, so I will release this.
If the first site root has the typoscript incldued indriectly as in our Projects, the change is not neccessery as the first SiteRoots Typoscript will always be loaded in the Backend Module:
It is like this since TYPO3 7 LTS
https://github.com/TYPO3/typo3/blob/main/typo3/sysext/extbase/Classes/Configuration/BackendConfigurationManager.php#L303
![grafik](https://github.com/pluswerk/mail_logger/assets/471387/6e5c7615-9983-4394-afd6-e2618a8f2402)

dimitri-koenig commented:
@Kanti, thx
